### ISSUE TITLE
разбить хостинги на категории для картинок и для файлов

### ISSUE THREAD

**Issue by arhangelsoft** [zenden2k/image-uploader#128]
Доброго времени!

Конкретно достал зоопарк длинного меню с выбором типа хостинга :) 

Может разбить на группы виде доп. пунктов меню "Хостинг файлов", "хостинг изображений" ?

**Comment by zenden2k** [comment]
А разбиение на колонки не помогло?

**Comment by arhangelsoft** [comment]
нет. Когда пользователь хочет грузить файлы, он видит только файловые сервисы и сервисы которые грузят все.
Когда пользователь грузит картинки, ему отображаются вообще все сервисы. До кучи они там ещё и сортируются по алфавиту. Поди пойми куда грузить именно картинку, да так чтоб в хостинг файлов не вляпаться.

**Comment by zenden2k** [comment]
У меня была идея, чтобы пользователи могли оценивать хостинги (ставить звездочки)

**Comment by zenden2k** [comment]
Что-то вроде
![clipboard2.png](http://goo.gl/Z0b5h6)

**Comment by arhangelsoft** [comment]
если совсем круто, то не меню с кнопками, а панель с выпадающими ListView(группами).
В ListView колонки:
http:// | рейтинг пользователей(звездочки от 1 до 5, если не голосовал, иначе текущие голоса)

сортировка соответственно от самых популярных до менее популярных.

Группы 2(для картинок, для всех файлов(можно скрыть)), либо одна но с фильтром по типу хостинга(тип: любые файлы, для изображений, для файлы).

На счёт рейтинга, можно ещё БД хостингов оформить, с рейтингом и отзывами пользователей IU.

**Comment by arhangelsoft** [comment]
Хостинг могу подсказать нормальный. hts.ru цены радуют, сервера не падают, тех. поддержка всегда адекватна и видимо никогда не спят (писал им в 3 утра...). Проплатил на 3 месяца вперед и забыл.

Есть тест-драйв на 10 дней без оплаты.

**Comment by zenden2k** [comment]
Вот как сделано в FileUploader, правда это статическая таблица, ничего выбрать нельзя
![screenshot 2015-04-23 001.png](https://lh3.googleusercontent.com/-YkjF_cpiFI8/VTi1XIDqbuI/AAAAAAAAEuM/CsiZJwiiGp4/s0/screenshot%2525202015-04-23%252520001.png)

**Comment by arhangelsoft** [comment]
Ну вот что-то типо этого, да. По нажатию выбора хостинга юзер видит ссылку, рейтинг пользователей, ссылку на страницу отзывов, конками отображаются нужды, а под таблицей легенда.
